pig/src/main/resources/interpreter-setting.json

```diff
@@ -15,6 +15,12 @@
         "propertyName": "zeppelin.pig.includeJobStats",
         "defaultValue": "false",
         "description": "flag to include job stats in output"
+      },
+      "zeppelin.pig.execType": {
+        "envName": "SPARK_MASTER",
+        "propertyName": "SPARK_MASTER",
```

Comment:
hmm, pretty sure the idea is the user only need to set of the environment, or the property if listed like this.
for instance https://github.com/apache/zeppelin/blob/master/spark/src/main/resources/interpreter-setting.json#L75
```
"zeppelin.spark.concurrentSQL": {
        "envName": "ZEPPELIN_SPARK_CONCURRENTSQL",
        "propertyName": "zeppelin.spark.concurrentSQL",
```
if `ZEPPELIN_SPARK_CONCURRENTSQL` is set but property `zeppelin.spark.concurrentSQL` is not, then it will be used as `zeppelin.spark.concurrentSQL`

so if someone sets `zeppelin.pig.execType` to `tez` and `SPARK_MASTER` to `local`?